src/modules/endpoint/sagas/loadCityContent.js

```diff
@@ -36,6 +38,12 @@ export default function * loadCityContent (
 
   if (!criterion.peeking()) {
     const appSettings = new AppSettings()
+    const previousSelectedCity = yield call(appSettings.loadSelectedCity)
+    const previousContentLanguage = yield call(appSettings.loadContentLanguage)
+    try {
+      yield NotificationsManager.unSubscribeToPreviousCityTopic(previousSelectedCity, previousContentLanguage)
+      yield NotificationsManager.subscribeToNewCityTopic(newCity, newLanguage)
+    } catch (e) { console.debug(e) }
```

Comment:
```suggestion
    } catch (e) { console.error(e) }
```

`error` is more appropriate here